k8s/cftptl/common/slack-help-bot/slack-help-bot.yaml

```
@@ -0,0 +1,37 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: slack-help-bot
+  namespace: admin
+  annotations:
+    fluxcd.io/automated: "true"
+    fluxcd.io/tag: glob:*
```

glob.* doesn't look like a good idea to me. 
```suggestion
    fluxcd.io/tag.nodejs: glob:*
```